Source/RageCommunity.Library/Graphics/Checkpoint.cs

```diff
@@ -0,0 +1,223 @@
+﻿using System;
+using System.Drawing;
+using Rage;
+using Rage.Native;
+using RageCommunity.Library.Extensions;
+
+namespace RageCommunity.Library.Graphics
+{
+    /// <summary>
+    /// This class represents a checkpoint within GTA V.
+    /// </summary>
+    /// <remarks>
+    /// Originally created by PNWParksFan. 
+    /// </remarks>
+    public class Checkpoint : IDeletable, ISpatial
+    {
+        private readonly Boolean setToGround;
```

Comment:
Do we want to follow the naming convention where private fields are prefaced by an underscore (_setToGround)?

Reply:
I suggest because this C# we stick to the Microsoft guidelines and do not use any kind of field prefix or underscore. 
https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/ta31s3bc(v=vs.71)?redirectedfrom=MSDN
Quote: `Do not apply a prefix to field names or static field names. Specifically, do not apply a prefix to a field name to distinguish between static and nonstatic fields. For example, applying a g_ or s_ prefix is incorrect.`

Reply:
I've read a bit of discussions, and it seems like the community is split on whether to use underscores or `this`.  Since we're the only two contributors right now, and we're also split on what should be used, I think we should leave it alone for now and use what we prefer.

Reply:
I agree. As long as we don't use underscores in public members it's fine for me that everyone uses their own style within classes. I will resolve this comment.